v03_pipeline/lib/reference_data/gencode/mapping_gene_ids.py

```diff
@@ -129,3 +132,16 @@ def load_gencode(gencode_release, download_path=''):
 
     logger.info(f'Got {len(gene_id_mapping)} gene id mapping records')
     return gene_id_mapping
+
+def load_gencode_ensembl_to_refseq_id(gencode_release: int):
+    url = GENCODE_ENSEMBL_TO_REFSEQ_URL.format(gencode_release=gencode_release)
+    response = requests.get(url, stream=True)
+    ensembl_to_refseq_ids = {}
+    for line in gzip.GzipFile(fileobj=response.raw):
+        line = line.decode('ascii').strip().split('\t')
+        if len(line) > 3:
+            raise ValueError(
+                'Unexpected number of fields on line in ensemble_to_refseq mapping',
+            )
+        ensembl_to_refseq_ids[line[0].split('.')[0]] = line[1]
```

Comment:
The mapping structure mirrors 'VEP': no minor version for ensembl, included minor version for refseq.